commands/user.go

```diff
@@ -104,30 +114,47 @@ func init() {
 	RootCmd.AddCommand(UserCmd)
 }
 
-func deactivateUsers(c client.Client, userArgs []string) {
+func userActivateCmdF(c client.Client, command *cobra.Command, args []string) error {
+	if len(args) < 1 {
+		return errors.New("Expected at least one argument. See help text for details.")
+	}
```

Comment:
This check can be replaced using `Args: cobra.MinimumNArgs(1),` in the command struct